### PR TITLE
Enhanced Documentation for Self-Hosting DocsGPT on Amazon Lightsail

### PR DESCRIPTION
**Pull request by rahul0x00**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update
- **Why was this change needed?** (You can also link to an open issue here)
This change was needed to improve the clarity, structure, and user-friendliness of the self-hosting guide for DocsGPT on Amazon Lightsail. The previous documentation was revised to address user feedback and provide a more comprehensive and informative guide for self-hosting.
- **Other information**:

### CODE REVIEW

**Comment by rahul0x00** [comment]
It looks more clean now

image

File: docs/pages/Deploying/Hosting-the-app.md [hunk 1]
```diff
@@ -6,47 +6,40 @@ Here's a step-by-step guide on how to setup an Amazon Lightsail instance to host
 
 (If you know how to create a Lightsail instance, you can skip to the recommended configuration part by clicking here).
 
-### 1. Create an account or login to https://lightsail.aws.amazon.com
+### 1. Create an AWS Account: 
+If you haven't already, create or log in to your AWS account at lightsail.aws.amazon.com.
```
Comment (dartpain) [inline docs/pages/Deploying/Hosting-the-app.md:10]: Please make it a clickable link

File: docs/pages/Deploying/Hosting-the-app.md [hunk 2]
```diff
@@ -6,47 +6,40 @@ Here's a step-by-step guide on how to setup an Amazon Lightsail instance to host
...
+PS: It may take a few minutes for the instance setup to complete.
 
-#### The recommended configuration is as follows:
+### Connecting to Your newly created Instance
 
-- Ubuntu 20.04 LTS
-- 1GB RAM
-- 1vCPU
-- 40GB SSD Hard Drive
-- 2TB transfer
```
Comment (dartpain) [inline docs/pages/Deploying/Hosting-the-app.md:31]: You need to make sure you specify that its ubuntu somewhere

File: docs/pages/Deploying/Hosting-the-app.md [hunk 3]
```diff
@@ -6,47 +6,40 @@ Here's a step-by-step guide on how to setup an Amazon Lightsail instance to host
...
 
 #### Download the package information
 
-Once it has finished cloning the repository, it is time to download the package information from all sources. To do so, simply enter the following command:
+Once it has finished cloning the repository, it is time to download the package information from all sources. To do so simply enter the following command:
 
 `sudo apt update`
 
-#### Install Docker and Docker Compose
+#### Installing Docker and Docker Compose
```
Comment (dartpain) [inline docs/pages/Deploying/Hosting-the-app.md:42]: Just leave install

File: docs/pages/Deploying/Hosting-the-app.md [hunk 4]
```diff
@@ -6,47 +6,40 @@ Here's a step-by-step guide on how to setup an Amazon Lightsail instance to host
...
-
-A terminal window will pop up, and the first step will be to clone the DocsGPT git repository:
+A terminal window will pop up, and the first step will be to clone the DocsGPT Git repository:
 
 `git clone https://github.com/arc53/DocsGPT.git`
 
 #### Download the package information
 
-Once it has finished cloning the repository, it is time to download the package information from all sources. To do so, simply enter the following command:
+Once it has finished cloning the repository, it is time to download the package information from all sources. To do so simply enter the following command:
```
Comment (dartpain) [inline docs/pages/Deploying/Hosting-the-app.md:38]: Leave that comma there i think its good

**Review by dartpain — CHANGES_REQUESTED** [review]
Thank you for this PR, I left a few comments

File: docs/pages/Deploying/Hosting-the-app.md (hunk 2 above)
Comment (rahul0x00) [reply docs/pages/Deploying/Hosting-the-app.md:31]: Hi, i've mentioned this in point c

image

should i specify it again?

File: docs/pages/Deploying/Hosting-the-app.md (hunk 4 above)
Comment (rahul0x00) [reply docs/pages/Deploying/Hosting-the-app.md]: done

**Comment by rahul0x00** [comment]
@dartpain
Hi, sorry to ping you but can you review the changes?

**Review by dartpain — APPROVED** [review]

**Comment by dartpain** [comment]
Sorry for a delay,
Thank you @holopin-bot @rahul0x00

**Comment by rahul0x00** [comment]
> Sorry for a delay, Thank you @holopin-bot @rahul0x00

Thanks for the badge, not gonna lie,the badge looks cool

**Pull request merged**